### ISSUE TITLE
[Feature Request] Option to disable teamspeak notifications when in-game

### ISSUE THREAD

**Issue by SekiAlpha** [michail-nikolaev/task-force-arma-3-radio#1540]
It would be nice to have an option to disable teamspeak notifications when in-game. I think it would also be a good idea to have this option enabled by default since notifications should only be useful for admins, not the common player.

**Comment by severgun** [comment]
I don't think that there is API to control client notification settings.

**Comment by SekiAlpha** [comment]
That may be so, however, ACRE does it somehow. I thought it would be a good feature to add to TFAR.

**Comment by dedmen** [comment]
yeah I also wanted to do this but didn't find a way in the API, but maybe I just didn't look hard enough.
And if needed I could always hack it
